iphone/Classes/TiUIListSectionProxy.m

```diff
@@ -137,11 +139,52 @@ - (void)setItems:(id)args withObject:(id)properties
   } else {
     [theDispatcher dispatchUpdateAction:^(UITableView *tableView) {
       [_items setArray:items];
+      [self rememberProxiesInItems:_items];
       [tableView reloadSections:[NSIndexSet indexSetWithIndex:_sectionIndex] withRowAnimation:animation];
     }];
   }
 }
 
+- (void)rememberProxiesInItems:(NSArray<NSDictionary *> *)items
+{
+  for (NSDictionary *item in items) {
+    for (NSString *key in item) {
+      if ([key isEqualToString:@"template"] || [key isEqualToString:@"properties"]) {
+        continue;
+      }
+      id itemPropertyValue = item[key];
+      if ([itemPropertyValue isKindOfClass:NSDictionary.class]) {
+        for (NSString *bindPropertyName in itemPropertyValue) {
+          id propertyValue = itemPropertyValue[bindPropertyName];
+          if ([propertyValue isKindOfClass:TiProxy.class]) {
+            [self rememberProxy:propertyValue];
+          }
+        }
+      }
+    }
+  }
+}
+
+- (void)forgetProxiesInItems:(NSArray<NSDictionary *> *)items
+{
+  for (NSDictionary *item in items) {
+    for (NSString *key in item) {
+      if ([key isEqualToString:@"template"] || [key isEqualToString:@"properties"]) {
+        continue;
+      }
+      id itemPropertyValue = item[key];
+      if ([itemPropertyValue isKindOfClass:NSDictionary.class]) {
+        for (NSString *bindPropertyName in itemPropertyValue) {
+          id propertyValue = itemPropertyValue[bindPropertyName];
+          if ([propertyValue isKindOfClass:TiProxy.class]) {
+            [self forgetProxy:propertyValue];
```

Comment:
Once validated: You could reuse the loop for both remember/forget to prevent duplicate code